index.js

```diff
@@ -17,28 +17,39 @@ function getConfigValue({ configName, defaultValue, env, input }) {
 
 module.exports = {
     onPreBuild: ({ utils, inputs }) => {
-      const expression = getConfigValue({
-        configName: 'expression',
-        defaultValue: '* * * * *',
-        env: process.env.DEPLOYMENT_HOURS_EXPRESSION,
-        input: inputs.expression
-      });
 
-      const timezone = getConfigValue({
-        configName: 'timezone',
-        defaultValue: 'America/Toronto',
-        env: process.env.DEPLOYMENT_HOURS_TIMEZONE,
-        input: inputs.timezone
+      const productionOnly = getConfigValue({
+        configName: 'productionOnly',
+        defaultValue: false,
+        env: process.env.DEPLOYMENT_HOURS_PRODUCTION_ONLY,
```

Comment:
Unfortunately, `process.env.DEPLOYMENT_HOURS_PRODUCTION_ONLY` is a string and not a boolean. I am open to only supporting this as a config defined in `manifest.yml` without the environment variable override.

Thoughts?